results/benchmark4h.1.yaml

```diff
@@ -0,0 +1,61 @@
+---
+benchmark_id: 4h.1
+line_plots:
+  - name: "all_data"
+    layout:
+      x_label: 'Fictive Time, <i>t</i><sub>aim</sub>&nbsp;[a.u.]'
```

Comment:
"Fictive" feels haughty. Can we say "Time" as in BM8, or "Simulation Time", or something people won't get confusing search results for to confirm the meaning?

Reply:
Happy to change it, but that terminology is used in the paper, see fig 5 in [paper](https://www.sciencedirect.com/science/article/pii/S092702561830168X)

Reply:
Curses! A perfect defense.